lib/core/src/lib/common/utils/date-fns-utils.ts

```diff
@@ -203,18 +210,19 @@ export class DateFnsUtils {
         return new Date(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate(), date.getUTCHours(), date.getUTCMinutes(), date.getUTCSeconds());
     }
 
-    static forceLocal(date: string | Date): Date {
-        if (typeof date === 'string'){
+    static forceLocal(date: Date | string): Date {
```

Comment:
`forceLocal` and `forceUtc` are meant to **display** the same date over different timezone.
So, giving a UTC date , the output might be a different Date. But since that date is localized automatically by the browser, the text displayed will be the same.
Eg: 03-24-1983T00:00:00Z localized to USA timezone is 03-23-1983T16:00:00 (notice the absence of Z, and different day and hours values). Instead forceUtc/forceLocal will convert the date to 03-23-1983T00:00:00, different value but same text when displayed